src/main/java/taiga/util/pbChange/PBChangeCalculator.java

```diff
@@ -0,0 +1,49 @@
+package taiga.util.pbChange;
```

Comment:
Since this isn't a taiga specific util, this should be moved to the `pbchange` package (`ui.metrics.pbchange`)

Reply:
Done